README.md

```diff
@@ -51,6 +51,10 @@ TIDY_ROOT = os.environ.get("TIDY_ROOT", "/User/johndoe/Downloads")  # "</path/to
 COIN_MARKET_CAP_API_KEY = os.environ.get("COIN_MARKET_CAP_API_KEY", "5i3qe43d-2de2-2eid-63b3-9403920395d2")
 DEFAULT_COIN = os.environ.get("DEFAULT_COIN", "BTC")
 CURRENCY = os.environ.get("CURRENCY", "GBP")
+
+# Calendar
+DEFAULT_EVENTS_NUMBER = os.environ.get("DEFAULT_EVENTS_NUMBER", "10")
+PATH_TO_CRED = os.environ.get("PATH_TO_CALENDAR_API_CRED", "/Users/john/cred/credentials.json")
```

Comment:
`DEFAULT_CALENDAR_EVENTS_NUMBER`?
`PATH_TO_CALENDAR_API_CRED` ?

Maybe change these to the above; `PATH_TO_CRED` sounded to vague and could get in the way of future cmds, I'm thinking

Reply:
Makes sense, will do